libres/lib/enkf/enkf_fs.cpp

```diff
@@ -176,7 +176,7 @@ struct enkf_fs_struct {
 UTIL_SAFE_CAST_FUNCTION(enkf_fs, ENKF_FS_TYPE_ID)
 UTIL_IS_INSTANCE_FUNCTION(enkf_fs, ENKF_FS_TYPE_ID)
 
-static void enkf_fs_umount(enkf_fs_type *fs);
+extern "C" void enkf_fs_umount(enkf_fs_type *fs);
```

Comment:
Can this be avoided here @dotfloat ?

Reply:
Iff. it's not used in Python, which I think it is.